plugins/module_utils/disk.py

```diff
@@ -72,7 +76,6 @@ def __init__(
 
     def to_hypercore(self):
         return dict(
-            uuid=self.uuid,
```

Comment:
Looks safe to me. I'm not sure if to_hypercore is ever used to POST/PATCH to server. Integration tests for vm, vm_disk do pass?

Reply:
Yes, vm and vm_disk pass